### ISSUE TITLE
Time for Training on CIFAR-100 and Tiny-ImageNet

### ISSUE THREAD

**Issue by jiachangliu** [QinbinLi/MOON#6]
Hello, thanks for the good work. I'm trying to reproduce the results shown in the paper. Training on CIFAR-100 and Tiny-ImageNet seems to be very slow. I'm using Titan Xp. For CIFAR-100, it took 4 hours to train for 4 global epochs; the test accuracy right now is 7.1%. For Tiny-ImageNet, it took 3 hours to train 1 global epoch; the test accuracy right now is 0.5%. I followed the preprocessing steps you suggested. The command lines are exactly what you had on the GitHub project page. I also followed the hyperparameters outlined in the paper.

Here are the command lines I used:
```
python main.py --dataset=cifar100 --alg=moon --lr=0.01 --mu=1 --epochs=10 --comm_round=100 --n_parties=10 --partition=noniid --beta=0.5 --logdir='./logs/' --datadir='./data/'

python main.py --dataset=tinyimagenet --alg=moon --lr=0.01 --mu=1 --epochs=10 --comm_round=20 --n_parties=10 --partition=noniid --beta=0.5 --logdir='./logs/' --datadir='./data/'
```

Is it normal to take such a long time to train on CIFAR-100 and Tiny-ImageNet? Can I ask how long it took you to finish training on CIFAR-100 and Tiny-ImageNet?

**Comment by QinbinLi** [comment]
Hi @jiachangliu ,

I didn't test MOON on Titan Xp. If running on a V100 GPU, the time to train on CIFAR-100 and Tiny-ImageNet is about 30 minutes and 200 minutes per round, respectively. Thus, I think your reported running time is normal.

**Comment by jiachangliu** [comment]
@QinbinLi Thanks for your answer.
